dali/base/dasds.cpp

```diff
@@ -3755,6 +3760,8 @@ int CSDSTransactionServer::run()
                 // to decrement when handling is complete.
                 // NB: most transactions are handled asynchronously, so decrement will need to happen on another thread
 
+                pSdsRequestsCount->inc(1);
```

Comment:
This doesn't need to be atomic.  It also replaces msgCount, so that code can be removed.